documentation/API_documentation/API Readiness Checklist device-identifier.md

```diff
@@ -1,27 +1,28 @@
 # API Readiness Checklist
 
-Checklist for device-identifier v0.2.0-rc.1 in r1.2.
+Checklist for device-identifier v0.2.0 in r1.3.
```

Comment:
Filename should be device-identifier-API-Readiness-Checklist.md

Reply:
> Filename should be device-identifier-API-Readiness-Checklist.md

Done